src/odb/src/db/dbBlock.cpp

```diff
@@ -4258,4 +4281,26 @@ dbTech* dbBlock::getTech()
   return (dbTech*) block->getTech();
 }
 
+dbSet<dbMarkerGroup> dbBlock::getMarkerGroups()
+{
+  _dbBlock* block = (_dbBlock*) this;
+  return dbSet<dbMarkerGroup>(block, block->_marker_group_tbl);
+}
+
+dbMarkerGroup* dbBlock::findMarkerGroup(const char* name)
+{
+  _dbBlock* block = (_dbBlock*) this;
+  return (dbMarkerGroup*)block->_marker_group_hash.find(name);
+}
+
+void dbBlock::writeMarkerGroups(const std::string& file)
```

Comment:
@QuantamHD will want to have

void dbBlock::writeMarkerGroups(const std::string& file)
void dbBlock::writeMarkerGroups(const std::ostream& file)

Where the former calls the later.

Reply:
❤️